mobile/library/common/jni/jni_utility.h

```diff
@@ -76,7 +76,7 @@ jlongArray native_final_stream_intel_to_array(JniHelper& jni_helper,
  */
 jobject native_map_to_map(JniHelper& jni_helper, envoy_map map);
 
-jstring native_data_to_string(JniHelper& jni_helper, envoy_data data);
+LocalRefUniquePtr<jstring> native_data_to_string(JniHelper& jni_helper, envoy_data data);
```

Comment:
Too late for this PR, but as we're moving away from a "C style" API here, should we consider moving these functions over to Envoy's camelCaseStyle?

Reply:
Yeah, that's also my plan once we have migrated everything to `JniHelper`.